homework/index.js

```diff
@@ -30,14 +30,129 @@
     return elem;
   }
 
+  function createOptions(repositories, select) {
+    repositories.forEach((repository, index) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: index,
+      });
+    });
+  }
+
+  function addRow(repository, table, property) {
+    const tbody = createAndAppend('tbody', table);
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('th', tr, {
+      text: `${property.toUpperCase()}: `,
+    });
+    if (property !== 'name') {
+      createAndAppend('td', tr, {
+        text: `${repository[property]}`,
+        id: property,
+      });
+    } else {
+      const firstRow = createAndAppend('td', tr);
+      createAndAppend('a', firstRow, {
```

Comment:
The assignment states that the associated GitHub repo page should be opened in a **new browser tab**.